### ISSUE TITLE
[Handbook update] Missing instructions for using CSS classes for images

### ISSUE THREAD

**Issue by mihajlija** [GoogleChrome/web.dev#4797]
**Describe the bug**
Instructions with CSS classes for images (w-screenshot, w-figure--fullbleed etc.) are gone from the handbook and I think we should bring them back, provided they all still work.

@robdodson Can you confirm that the web.dev still supports all the classes it we used before switching to imgix? 

Reference: https://github.com/GoogleChrome/web.dev/commit/6ee2875fd4e3d83983ff6dc1565eed98914dfda1#diff-ce4aafaf56fca08fa9e82e760e98b26cb7729292dd83b104fd83439d715bebf0

**Comment by robdodson** [comment]
@mihajlija yep all of the classes are still supported. `{% Img %}` will output an `<img>` so anything that used to work on `<img>` should continue to work.

**Comment by robdodson** [comment]
@mihajlija, @MichaelSolati and @devnook have been doing some bug cleanup recently and I was thinking that if you weren't planning to work on this immediately I could reassign it to one of them. Would that be ok?

**Comment by rachelandrew** [comment]
@devnook I'm assuming this one is now redundant given the updates? Do we have classes like this in the new design system?

**Comment by devnook** [comment]
Yes, we can close this.
